routes/alfresco.py

```diff
@@ -396,47 +318,22 @@ def run_aggregate_var_polygon(var_ep, poly_gdf, poly_id):
     poly_mask_arr = get_poly_mask_arr(ds_list[0], poly, bandname)
     # average over the following decades / time periods
     aggr_results = {}
-    if cov_id_str in ["relative_flammability", "relative_vegetation_change"]:
-        historical_results = summarize_within_poly_marr(
-            ds_list[0],
-            poly_mask_arr,
-            flammability_historical_dim_encodings,
-            bandname,
-            varname,
-        )
-        #  add the model, scenario, and varname levels for CRU
-        for era in historical_results:
-            aggr_results[era] = {
-                "CRU-TS40": {"CRU_historical": {varname: historical_results[era]}}
-            }
-        # run regular future
+    if var_ep == "flammability":
         ar5_results = summarize_within_poly_marr(
-            ds_list[-1],
-            poly_mask_arr,
-            flammability_future_dim_encodings,
-            bandname,
-            varname,
+            ds_list[-1], poly_mask_arr, flammability_dim_encodings, bandname, var_ep
         )
-        for era, summaries in ar5_results.items():
-            aggr_results[era] = summaries
-        # run summary eras for future
-        summary_eras = ["2040-2069", "2070-2099"]
-        for ds, era in zip(ds_list[1:3], summary_eras):
-            aggr_results[era] = summarize_within_poly_marr(
-                ds, poly_mask_arr, flammability_future_dim_encodings, bandname, varname
-            )
-    elif cov_id_str == "alfresco_vegetation_type_percentage":
+    elif var_ep == "veg_type":
         ar5_results = summarize_within_poly_marr(
-            ds_list[-1], poly_mask_arr, veg_type_dim_encodings, bandname, varname
+            ds_list[-1], poly_mask_arr, veg_type_dim_encodings, bandname, var_ep
         )
-        for era, summaries in ar5_results.items():
-            aggr_results[era] = summaries
-        aggr_results = remove_invalid_dim_combos(aggr_results)
+    for era, summaries in ar5_results.items():
+        aggr_results[era] = summaries
+    aggr_results = remove_invalid_dim_combos(var_ep, aggr_results)
 
     return aggr_results
 
 
-def remove_invalid_dim_combos(results):
+def remove_invalid_dim_combos(var_ep, results):
```

Comment:
I like this idea a lot! This will help keep all metadata pertaining to the coverage in Rasdaman and make the API code more coverage agnostic and easier to generalize and recycle in the future. Good idea for future iteration.